adr/component_structure.md

```diff
@@ -0,0 +1,102 @@
+# Create both low-level and high-level components that are exported from each package.
+
+## Context
+
+The primary issue we are facing right now is determining how "high-level" the component should be. For example, should we create a single component with many props, or should we create many components to be composed together.
```

Comment:
Our 2 main goals with Clay3 are:
- Provide enough flexibility and low-level components for teams to quickly build and deliver applications without being blocked by us
- Provide quality high level components that allow teams to quickly build applications with the best possible interactions (UX) that are consistent, performant, accessible and extensible without the need of repeating themselves over and over again.